smart-cards-tests/pom.xml

```diff
@@ -106,6 +106,8 @@
                 <argument>-Dlogging.level.root=INFO</argument>
                 <argument>-Ddata-query.internal-url=http://localhost:8777</argument>
                 <argument>-Ddata-query.r4-base-path=/</argument>
+                <argument>-Djwk-set.private-json='{\"keys\":[{\"kty\":\"EC\",\"d\":\"5h-A-oFDpePWC9zZ54_sWANZlwDxppu2-DQuAuvW3Bw\",\"use\":\"sig\",\"crv\":\"P-256\",\"kid\":\"gSb6VludekqwZS_0osHNxAK2y2Rua1V_aGWNFaTvgNM\",\"x\":\"WcBuVJRGiCvhEdU6nbKysIxEjQOqzaH-kiIZoIpFNfE\",\"y\":\"FiWqBp6ilDBJ-VEnZsgHTK4fYKmB841n2wNMfvFZB2E\",\"alg\":\"ES256\"}]}'</argument>
```

Comment:
are the enclosing single-quotes necessary here?

Reply:
On mingw I did not need them. Jenkins' build was failing as Linux/bash messed with the quotes, so the json parser failed. This wrapping with single quotes, along with the `cleanupJson` method, solved that problem. I'm unsure if this is the correct solution, though.